javascript/javascript2/week2/quizapp.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Quiz App</title>
+  <link rel="stylesheet" href="styles.css">
```

Comment:
There is no CSS file with this name, most likely you renamed the file to `quizzapp.css` just before submitting the homework and forgot to replace it in code.